services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -88,6 +89,8 @@ export interface RateInfoFormType {
     rateDocuments: SubmissionDocument[]
     actuaryContacts: ActuaryContact[]
     actuaryCommunicationPreference?: ActuaryCommunicationType
+    packagesWithSharedRateCerts: string[]
+    sharedRateCert: boolean
```

Comment:
I find it's helpful to name variables that hold booleans to something that starts with "is" or "has," so we immediately  know we're dealing with a boolean.  `hasSharedRateCert`?